spynnaker7/pyNN/models/pynn_projection.py

```diff
@@ -1,3 +1,7 @@
+from spynnaker.pyNN.models.abstract_models import AbstractAcceptsIncomingSynapses
+from spynnaker.pyNN.models.neuron.synapse_dynamics import AbstractStaticSynapseDynamics
+from spynnaker.pyNN.models.neuron.synapse_dynamics.abstract_synapse_dynamics_structural import \
```

Comment:
I think this line can be shortened a little with some corresponding editing on the sPyNNaker side.